frontend/src/components/pod/List.tsx

```diff
@@ -111,3 +118,135 @@ export default function PodList() {
+  if (!!pod.status.reason) {
+    reason = pod.status.reason;
+  }
+
+  let initializing = false;
+  for (const i in pod.status.initContainerStatuses) {
+    const container = pod.status.initContainerStatuses[i];
+    restarts += container.restartCount;
+    if (!!container.lastState.terminated) {
+      const terminatedDate = new Date(container.lastState.terminated.finishedAt);
+      if (lastRestartDate.getTime() < terminatedDate.getTime()) {
+        lastRestartDate = terminatedDate;
+      }
+    }
+
+    switch (true) {
+      case !!container.state.terminated && container.state.terminated.exitCode === 0:
+        continue;
+      case !!container.state.terminated:
+        if (!container.state.terminated.reason) {
+          if (container.state.terminated.signal !== 0) {
+            reason = `Init:Signal:${container.state.terminated.signal}`;
+          } else {
+            reason = `Init:ExitCode:${container.state.terminated.exitCode}`;
+          }
+        } else {
+          reason = 'Init:' + container.state.terminated.reason;
+        }
+        initializing = true;
+        break;
+      case !!container.state.waiting &&
+        !!container.state.waiting.reason &&
+        container.state.waiting.reason !== 'PodInitializing':
+        reason = 'Init:' + container.state.waiting.reason;
+        initializing = true;
+        break;
+      default:
+        reason = `Init:${i}/${(pod.spec.initContainers || []).length}`;
+        initializing = true;
+    }
+    break;
+  }
+
+  if (!initializing) {
+    restarts = 0;
+    let hasRunning = false;
+    for (let i = pod.status.containerStatuses.length - 1; i >= 0; i--) {
+      const container = pod.status.containerStatuses[i];
+
+      restarts += container.restartCount;
+      if (!!container.lastState?.terminated) {
+        const terminatedDate = new Date(container.lastState.terminated.finishedAt);
+        if (lastRestartDate.getTime() < terminatedDate.getTime()) {
+          lastRestartDate = terminatedDate;
+        }
+      }
+
+      if (!!container.state.waiting && !!container.state.waiting.reason) {
+        reason = container.state.waiting.reason;
+      } else if (!!container.state.terminated && !!container.state.terminated.reason) {
+        reason = container.state.terminated.reason;
+      } else if (!!container.state.terminated && !container.state.terminated.reason) {
+        if (container.state.terminated.signal !== 0) {
+          reason = `Signal:${container.state.terminated.signal}`;
+        } else {
+          reason = `ExitCode:${container.state.terminated.exitCode}`;
+        }
+      } else if (container.ready && !!container.state.running) {
+        hasRunning = true;
+        readyContainers++;
+      }
+    }
+
+    // change pod status back to "Running" if there is at least one container still reporting as "Running" status
+    if (reason === 'Completed' && hasRunning) {
+      if (hasPodReadyCondition(pod.status.conditions)) {
+        alert('Running');
```

Comment:
This was probably left over from debugging?

Reply:
Yes, it is. Please delete it.